tstime/tstime.go

```diff
@@ -59,3 +59,35 @@ func Sleep(ctx context.Context, d time.Duration) bool {
 		return true
 	}
 }
+
+type Clock interface {
```

Comment:
Docstrings throughout please, on: interfaces, interface methods, structs and struct methods.